airflow/config_templates/config.yml

```diff
@@ -2591,3 +2591,22 @@ sensors:
       type: float
       example: ~
       default: "604800"
+scarf_analytics:
+  description: |
+    Airflow integrates `Scarf <https://about.scarf.sh/>`__ to collect basic telemetry data during operation.
+    This data assists Airflow maintainers in better understanding how Airflow is used.
+    Insights gained from this telemetry are critical for prioritizing patches, minor releases, and
+    security fixes. Additionally, this information supports key decisions related to the development Roadmap.
+
+    Users can easily opt out of analytics in various ways documented
+    `here <https://docs.scarf.sh/gateway/#do-not-track>`__ and by setting the ``enabled`` option
```

Comment:
Is this link right? We aren't planning to use the Scarf gateway?

Reply:
Hey @jedcunningham! I work on the Scarf team. 

Hmm, maybe I can help provide some clarity here, but I'm not exactly sure what the question is regarding the tracking pixels. 

Scarf does provide a tracking pixel, and they are separate from the Gateway (which, as you mentioned, does act as a proxy/redirect for things like container images, files, Python packages, etc.), but the data from the two mechanisms does get correlated in the Scarf platform. 

Does that help answer what you're asking? Let me know if I'm misinterpreting. Will definitely look to update our docs for further clarity based on this discussion :)

Reply:
Hey @arjundevarajan! Thanks for jumping in 🙏.

Looking again, I do see this sentence:

> Please note that this behavior works for all packages and pixels that are served through our gateway. If users have DNT turned on in their browser settings, we will not track file downloads or pixel views.

Sounds like pixels also go through the gateway then, right?

I missed that (or, it didn't quite register to me initially) the first time around. As someone completely new to Scarf, I opened that link, saw that DNT was being honored (cool), thought "huh, what is the Scarf gateway?", then jumped to the top of the page and saw:

> Scarf Gateway is a service that sits in front of your existing software hosting platform(s), acting as a single access-point to all of your artifacts, regardless of where they are actually hosted.

And boom, confused, since I knew we weren't integrating our downloads. I'd say the intro is too narrow since it sounds like the gateway also is the ingress point for pixels. That's not represented anywhere on that page other than the DNT section. Anyways, my 2c.

Reply:
>But maybe the tracking pixels also go through scarf?

That's correct Jed and the pixel we create will be routed via the gateway so that's should still be true I think

Reply:
Sorry, yeah, I meant to say "tracking pixels also go through scarf gateway".